archiver/direct.py

```diff
@@ -60,9 +60,11 @@ def direct_archiver_from_params(
     biosamples_converter = BioSamplesConverter(biosamples_domain)
     biosamples_submitter = BioSamplesSubmitter(biosamples_client, biosamples_converter)
 
-    biostudies_client = BioStudies(biostudies_url, biostudies_username, biostudies_password)
-    biostudies_converter = BioStudiesConverter()
-    biostudies_submitter = BioStudiesSubmitter(biostudies_client, biostudies_converter)
+    biostudies_submitter = None
+    # TODO when we solved the issue with BioStudies availability, then we have change this lines back
+    # biostudies_client = BioStudies(biostudies_url, biostudies_username, biostudies_password)
+    # biostudies_converter = BioStudiesConverter()
+    # biostudies_submitter = BioStudiesSubmitter(biostudies_client, biostudies_converter)
     return DirectArchiver(loader=hca_loader, updater=hca_updater,
```

Comment:
How would this constructor change when we support ENA?
When we add an additional archive, such as SCEA?
Seems there would be a few changes to this class: in the constructor, in the `__archive` function, in `direct_archiver_from_params`.
Seems to me this could be better separated per archive, so that adding or disabling an archive is done in a single location, rather than spread over several functions/classes/files.